unit_test/test_spdm_requester/get_measurement_extension_log.c

```diff
@@ -431,6 +492,179 @@ libspdm_return_t libspdm_requester_get_measurement_extension_log_test_receive_me
+                             LIBSPDM_MAX_MEL_BLOCK_LEN;
+
+        libspdm_transport_test_encode_message(spdm_context, NULL, false,
+                                              false, spdm_response_size,
+                                              spdm_response, response_size,
+                                              response);
+        calling_index++;
+    }
+        return LIBSPDM_STATUS_SUCCESS;
+    case 0x8:
+    {
+        spdm_measurement_extension_log_response_t *spdm_response;
+        size_t spdm_response_size;
+        size_t transport_header_size;
+        spdm_measurement_extension_log_dmtf_t *spdm_mel;
+
+        spdm_mel = (spdm_measurement_extension_log_dmtf_t *)m_libspdm_mel_test;
+
+        transport_header_size = LIBSPDM_TEST_TRANSPORT_HEADER_SIZE;
+        spdm_response = (void *)((uint8_t *)*response + transport_header_size);
+
+        spdm_response->header.spdm_version = SPDM_MESSAGE_VERSION_13;
+        spdm_response->header.request_response_code = SPDM_MEASUREMENT_EXTENSION_LOG;
+        spdm_response->header.param1 = 0;
+        spdm_response->header.param2 = 0;
+        /* Portion_length is greater than the LIBSPDM_MAX_MEL_BLOCK_LEN */
+        spdm_response->portion_length = LIBSPDM_MAX_MEL_BLOCK_LEN + 1;
+        spdm_response->remainder_length = 0;
+
+        libspdm_copy_mem(spdm_response + 1,
+                         (size_t)(*response) + *response_size - (size_t)(spdm_response + 1),
+                         (uint8_t *)spdm_mel,
+                         LIBSPDM_MAX_MEL_BLOCK_LEN);
+
+        spdm_response_size = sizeof(spdm_measurement_extension_log_response_t) +
+                             LIBSPDM_MAX_MEL_BLOCK_LEN;
+
+        libspdm_transport_test_encode_message(spdm_context, NULL, false,
+                                              false, spdm_response_size,
+                                              spdm_response, response_size,
+                                              response);
+    }
+        return LIBSPDM_STATUS_SUCCESS;
+
+    case 0x9:
+    {
+        spdm_measurement_extension_log_response_t *spdm_response;
+        size_t spdm_response_size;
+        size_t transport_header_size;
+        spdm_measurement_extension_log_dmtf_t *spdm_mel;
+
+        spdm_mel = (spdm_measurement_extension_log_dmtf_t *)m_libspdm_mel_test;
+
+        transport_header_size = LIBSPDM_TEST_TRANSPORT_HEADER_SIZE;
+        spdm_response = (void *)((uint8_t *)*response + transport_header_size);
+
+        spdm_response->header.spdm_version = SPDM_MESSAGE_VERSION_13;
+        spdm_response->header.request_response_code = SPDM_MEASUREMENT_EXTENSION_LOG;
+        spdm_response->header.param1 = 0;
+        spdm_response->header.param2 = 0;
+        spdm_response->portion_length = LIBSPDM_MAX_MEL_BLOCK_LEN;
+        /* remainder_length is greater than the SPDM_MAX_MEASUREMENT_EXTENSION_LOG_SIZE */
+        spdm_response->remainder_length = SPDM_MAX_MEASUREMENT_EXTENSION_LOG_SIZE + 1;
```

Comment:
`SPDM_MAX_MEASUREMENT_EXTENSION_LOG_SIZE + 1` is `0`.

Reply:
Thanks, I have already modified these problems.